code/datums/supplypacks.dm

```diff
@@ -885,6 +885,20 @@ var/list/all_supply_groups = list(supply_emergency,supply_security,supply_engine
 	cost = 60
 	containername = "Pizza crate"
 
+/datum/supply_packs/organic/condiments
+        name = "Condiments crate"
+        contains = list(/obj/item/weapon/reagent_containers/food/condiment/coldsauce,
+                                        /obj/item/weapon/reagent_containers/food/condiment/hotsauce,
+                                        /obj/item/weapon/reagent_containers/food/condiment/tasteenchancer,
+                                        /obj/item/weapon/reagent_containers/food/condiment/sugar,
+                                        /obj/item/weapon/reagent_containers/food/condiment/soysauce,
+                                        /obj/item/weapon/reagent_containers/food/condiment/ketchup,
+                                        /obj/item/weapon/reagent_containers/food/condiment/peppermill,
+                                        /obj/item/weapon/reagent_containers/food/condiment/saltshaker)
+       cost = 20
```

Comment:
Compile error says you've got inconsistent indentation somewhere around this line. Double-check to make sure you didn't accidentally throw a space somewhere in the tabbing. This should all be coming up on your Dreammaker output when you compile your changes in it. If you aren't making the changes in DreamMaker or are not compiling them, please start doing so.